experimental/ModelBuilder/test/CMakeLists.txt

```
@@ -0,0 +1,275 @@
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+iree_add_all_subdirs()
+
+file(GLOB _GLOB_X_CPP LIST_DIRECTORIES false RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} CONFIGURE_DEPENDS *.cpp)
+file(GLOB _GLOB_BENCHX LIST_DIRECTORIES false RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} CONFIGURE_DEPENDS Bench*)
+list(REMOVE_ITEM _GLOB_X_CPP ${_GLOB_BENCHX})
+iree_lit_test_suite(
+  NAME
+    lit
+  SRCS
+    "${_GLOB_X_CPP}"
+  DATA
+    test-dot-prod
+    test-matmul-vulkan
+    test-mnist-jit
+    test-simple-jit
+    test-simple-jit-vulkan
+    test-simple-mlir
+    test-vec-to-gpu
+    test-vector-transfers-jit
+    iree::tools::IreeFileCheck
+)
+
+iree_cc_binary(
+  NAME
+    test-dot-prod
+  OUT
+    test-dot-prod
+  SRCS
+    "TestDotProdJIT.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-vector-transfers-jit
+  OUT
+    test-vector-transfers-jit
+  SRCS
+    "TestVectorTransfersJIT.cpp"
+  DEPS
+    runtime-support.so
+    LLVMSupport
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-mnist-jit
+  OUT
+    test-mnist-jit
+  SRCS
+    "TestMNISTJIT.cpp"
+  DEPS
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    MLIRmlir_runner_utils
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-simple-jit
+  OUT
+    test-simple-jit
+  SRCS
+    "TestSimpleJIT.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-simple-jit-vulkan
+  OUT
+    test-simple-jit-vulkan
+  SRCS
+    "TestSimpleJITVulkan.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIRIR
+    MLIRParser
+    MLIRSPIRV
+    MLIRmlir_runner_utils
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+    iree::base::initializer
+    iree::hal::llvmjit::llvmjit_driver_module
+    iree::hal::vmla::vmla_driver_module
+    iree::hal::vulkan::vulkan_driver_module
+    vulkan-runtime-wrappers
+)
+
+iree_cc_binary(
+  NAME
+    test-matmul-vulkan
+  OUT
+    test-matmul-vulkan
+  SRCS
+    "TestMatMulVulkan.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIRExecutionEngine
+    MLIRGPU
+    MLIRGPUToSPIRVTransforms
+    MLIRGPUToVulkanTransforms
```

Here it the `To` is capitalized, where in other CMake deps it is not. So either we wait for the LLVM bump and change all to `MLIRGPUToVulkanTransforms` or we need to fix this one and a few others.

```suggestion
MLIRGPUtoVulkanTransforms
```

---

Done.